### PR TITLE
Feature/new filter form graph kory h

### PR DESCRIPTION
**Pull request by Tusk617**
## Solution description

Describe the code change in detail for reviewer. Explain the technical solution you have provided and how it fixed the issue case.

## Covered unit test cases

Were unit test cases recorded for this fix?

## Code is clear and maintainable:

- [x] My code follows the style guidelines of this project
- [x] My code is consistently formatted.
- [x] No instance on duplicate code.

## Tested this issue fix on all browser

[x] Chrome
[x] Safari
[x] Internet Explorer
[x] Firefox

## Keep Pull Request Simple

Pull Request is relatively small. It does not contain multiple changes or solve multiple problems.

## Reviewed by Team members

Pull Request is reviewed by one or more team members

## Idiomatic Code

Code follows industry standards for language, framework, and libraries.

### CODE REVIEW

File: src/components/common/Graph.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
 import React, { useEffect, useState } from 'react';
 import { useSelector } from 'react-redux';
-
+//Alexander changes
```
Comment (schroeder-g) [inline src/components/common/Graph.js]: Not sure that we need these comments? I don't think its a huge deal but github can make this kind of attribution for us

**Review by schroeder-g — APPROVED** [review]
Nice work with the filter, Kory.

File: src/components/common/Graph.js (hunk 1 above)
Comment (bneil7) [reply src/components/common/Graph.js]: ^agreed, we could delete these through github directly without another commit push

**Review by bneil7 — APPROVED** [review]
solid work on implementing that new component

**Pull request merged**